Kernel/Storage/SD/Commands.cpp

```diff
@@ -0,0 +1,194 @@
+    cmd.response_type = ResponseType::ResponseOf136Bits;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd3()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::SendRelativeAddr;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd6()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::AppSetBusWidth;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+
+    return cmd;
+}
+
+constexpr Command build_cmd7()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::SelectCard;
+    cmd.response_type = ResponseType::ResponseOf48BitsWithBusy;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd8()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::SendIfCond;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd9()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::SendCsd;
+    cmd.response_type = ResponseType::ResponseOf136Bits;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd16()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::SetBlockLen;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd17()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::ReadSingleBlock;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.is_data = true;
+    cmd.crc_enable = true;
+    cmd.direction = DataTransferDirection::CardToHost;
+
+    return cmd;
+}
+
+constexpr Command build_cmd18()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::ReadMultipleBlock;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.is_data = true;
+    cmd.crc_enable = true;
+    cmd.direction = DataTransferDirection::CardToHost;
+    // 1.11 Auto CMD12
+    // The Host Driver should set Auto CMD12 Enable in the Transfer
+    // Mode register when issuing a multiple block transfer command
+    cmd.auto_command = SendAutoCommand::Command12;
+    cmd.block_counter = true;
+    cmd.multiblock = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd24()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::WriteSingleBlock;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.is_data = true;
+    cmd.crc_enable = true;
+
+    return cmd;
+}
+
+constexpr Command build_cmd25()
+{
+    Command cmd = {};
+    cmd.index = CommandIndex::WriteMultipleBlock;
+    cmd.response_type = ResponseType::ResponseOf48Bits;
+    cmd.is_data = true;
+    cmd.crc_enable = true;
+    // 1.11 Auto CMD12
+    // The Host Driver should set Auto CMD12 Enable in the Transfer
+    // Mode register when issuing a multiple block transfer command
+    cmd.auto_command = SendAutoCommand::Command12;
+    cmd.block_counter = true;
+    cmd.multiblock = true;
```

Comment:
(is this something extractable?)

Reply:
There are only two commands that set those bits this way, feels overkill to extract a function just for that.